openrtb_ext/imp_sovrnXsp.go

```diff
@@ -0,0 +1,8 @@
+package openrtb_ext
+
+type ExtImpSovrnXsp struct {
+	PubID    string `json:"pub_id,omitempty"`
```

Comment:
static/bidder-params/sovrnXsp.json marks `PubID` as required field. So cannot be omitempty

Reply:
For now `pub_id` is required; however, we're making changes to allow *either* `pub_id` or `zone_id` to be specified. So for the next update to this adapter, I'd like to keep this as-is, once the backend changes on our end are complete, the bidder params will be updated to require `pub_id` conditionally.

Reply:
> For now `pub_id` is required; however, we're making changes to allow _either_ `pub_id` or `zone_id` to be specified. So for the next update to this adapter, I'd like to keep this as-is, once the backend changes on our end are complete, the bidder params will be updated to require `pub_id` conditionally.

Would suggest to add omit empty when  _either_ `pub_id` or `zone_id` is allowed to specify. Untill then lets be inline with required fields mentioned in static/bidder-params/sovrnXsp.json.  Therefore, should remove `omitempty` for `PubID`. The omit empty  tag can be added in separate PR when backend changes on your end are complete, and  bidder params will be updated to require pub_id conditionally.

Reply:
Got it. Will make this change.